app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".AuthActivity">
```

Comment:
This raises a question for me: how do we handle the main activity differently for logged-in users and non-logged-in ones?

Reply:
yes, this also raise that question for me. Currently looking for answer.

Reply:
Maybe we can ask dr dx tmr